entities/campaignschedules.go

```diff
@@ -13,3 +13,7 @@ type CampaignSchedule struct {
 	CreatedAt   time.Time   `json:"created_at"`
 	UpdatedAt   time.Time   `json:"updated_at"`
 }
+
+func (cs *CampaignSchedule) TableName() string {
+	return "scheduled_campaigns"
```

Comment:
The name seems okay here, why should we change it?

Reply:
My bad I have an old version where the table was named `scheduled_campaigns`. I am closing this PR.